test/general/build_targets/package_test.dart

```diff
@@ -430,11 +430,12 @@ void _installFakeEngineArtifacts(Directory engineArtifactDir) {
     engineArtifactDir.childDirectory(directory).createSync(recursive: true);
   }
   for (final String file in <String>[
-    'tizen-common/icu/icudtl.dat',
+    'tizen-arm/5.5/libflutter_tizen_common.so',
+    'tizen-arm-debug/icudtl.dat',
     'tizen-arm-debug/libflutter_engine.so',
-    'tizen-arm-debug/libflutter_tizen_common.so',
+    'tizen-arm/5.5/libflutter_tizen_common.so',
```

Comment:
There are the same content in the 433 line.

Reply:
Fixed!